dataland-frontend/src/utils/FileUploadUtils.ts

```diff
@@ -0,0 +1,86 @@
+import { CompanyReport } from "@clients/backend";
+import { ApiClientProvider } from "@/services/ApiClients";
+import Keycloak from "keycloak-js";
+
+export interface DocumentToUpload {
```

Comment:
I find it a bit confusing that sometimes the files are called "file" and in other cases "document".
For example here we have "DocumentToUpload", "StoredReport", "ReportToUpload" and "uploadFiles". Is there a reason for this?

Reply:
ok here we agreed to have a team meeting to discuss the file hierachy. We want to make sure that this is applicable to all other cases in which we want to use documents. Further it should be discussed how to refactor the relevant parts in eu taxonomy.

Reply:
We discussed that we want this structure also for other occurences of reports. We create a new story for this.